twitter_ads/audience.py

```diff
@@ -183,3 +185,67 @@ def delete(self):
 resource_property(TailoredAudiencePermission, 'tailored_audience_id')
 resource_property(TailoredAudiencePermission, 'granted_account_id')
 resource_property(TailoredAudiencePermission, 'permission_level')
+
+
+class AudienceIntelligence(Resource):
+
+    PROPERTIES = {}
+
+    RESOURCE_BASE = '/3/accounts/{account_id}/audience_intelligence/'
+    RESOURCE_CONVERSATIONS = RESOURCE_BASE + 'conversations'
+    RESOURCE_DEMOGRAPHICS = RESOURCE_BASE + 'demographics'
+    METHOD = 'post'
+    HEADERS = {'Content-Type': 'application/json'}
+
+    @classmethod
+    def __get(klass, account, client, params):
+        """
+        Helper function to get the conversation data
+        Returns a Cursor instance
+        """
+        resource = klass.RESOURCE_CONVERSATIONS.format(account_id=account.id)
+
+        request = Request(
+            account.client, klass.METHOD,
+            resource, headers=klass.HEADERS, body=params)
+        return Cursor(klass, request, init_with=[account])
+
+    def conversations(self):
+        """
+        Get the conversation topics for an input targeting criteria
+        """
+        body = {
+            "conversation_type": self.conversation_type,
+            "audience_definition": self.audience_definition,
+            "targeting_inputs": self.targeting_inputs
+        }
+        return self.__get(account=self.account, client=self.account.client, params=json.dumps(body))
```

Comment:
Do we need to use `json.dumps` here? If `body` is a `dict`, it's my understanding that it should work.

Reply:
yep, i thought so too but it turns out these are all stored as "property" objects, so i had to convert them to JSON to avoid this:
```twitter_ads.error.BadRequest: <BadRequest object at 0x10d69c050 code=400 details=[{u'message': u'The JSON string format is invalid', u'code': u'INVALID'}]>```